components/ui/addToCalendar/CalendarButton.tsx

```
@@ -1,7 +1,11 @@
 import { memo } from "react";
 import { PlusIcon } from "@heroicons/react/outline";
+import { CalendarButtonProps } from "types/common";
 
-const CalendarButton = ({ onClick, hideText = false }) => (
+const CalendarButton: React.FC<CalendarButtonProps> = ({
```

ESLint

[error] 'React' is not defined.

(no-undef)